foreman.toml

```diff
@@ -1,4 +1,5 @@
 [tools]
-rojo = { source = "rojo-rbx/rojo", version = "6.2.0" }
-selene = { source = "Kampfkarren/selene", version = "0.14" }
-stylua = { source = "JohnnyMorganz/StyLua", version = "0.10.1" }
+rojo = { source = "rojo-rbx/rojo", version = "7.2.1" }
+selene = { source = "Kampfkarren/selene", version = "0.20.0" }
+stylua = { source = "JohnnyMorganz/StyLua", version = "0.14.2" }
+luau-lsp = { source = "JohnnyMorganz/luau-lsp", version = "*" }
```

Comment:
Worth noting that foreman will always use the newest available version in CI when `*` is the version specified; it might be better to use something like "1.8.1", which defaults to the `^` rule (keep using the latest 1.x, but don't use 2.x)

Reply:
done!